Server content/Resources/Prototypes/Roles/Jobs/Engineering/builder.yml

```
@@ -0,0 +1,35 @@
+- type: job
+  id: Builder
+  name: job-name-builder
+  description: job-description-builder
```

```suggestion
  description: Build, build, build!
```
Нету Американ локализации описания роли